doc/modules/manifold.rst

```diff
@@ -457,6 +460,19 @@ order to avoid that, the disparities :math:`\hat{d}_{ij}` are normalized.
    :align: center
    :scale: 60
 
+.. _multidimensional_scaling_method:
+
+Method
+-------------
+
+Metric :class:`MDS` offers two different algorithms (methods) to calculate
+results: SMACOF and SVD-based. The SMACOF method (Scaling by MAjorizing a
+COmplicated Function) minimizes objective function (stress) in iterative
```

Comment:
```suggestion
COmplicated Function) minimizes its objective function (stress) in an iterative
```

Reply:
> 
The 'O' after 'C' on "Complicated" is capitalized. Im sure that was by accident

Reply:
It was intentional, @zoj613, as it demonstrates where the abbreviation SMACOF derives from

Reply:
Done